repos/fdbt-site/src/pages/api/chooseValidity.ts

```diff
@@ -0,0 +1,65 @@
+import { NextApiResponse } from 'next';
+import { NextApiRequestWithSession, ErrorInfo } from '../../interfaces/index';
+import { updateSessionAttribute } from '../../utils/sessions';
+import { DURATION_VALID_ATTRIBUTE } from '../../constants/attributes';
+import { redirectToError, redirectTo } from './apiUtils';
+import { isValidInputDuration } from './apiUtils/validator';
+
+export const isInvalidValidityNumber = (validityInput: number): boolean => {
+    if (Number.isNaN(validityInput)) {
+        return true;
+    }
+
+    if (!Number.isInteger(Number(validityInput))) {
+        return true;
+    }
+
+    if (validityInput > 1000 || validityInput < 1) {
+        return true;
+    }
+
+    return false;
+};
+
+export const isInvalidInput = (validityInput: string): boolean => {
+    if (!validityInput || validityInput === '0' || isInvalidValidityNumber(Number(validityInput))) {
+        return true;
+    }
+    return false;
+};
```

Comment:
Should these also live in the validation file rather than in here?

Reply:
@RobCatton as it only gets used in choose validity is it ok to stay where it is?

Reply:
No, I think it makes sense to move them.